src/components/ModalPayment/ModalCardDetails/ModalCardDetails.tsx

```diff
@@ -81,6 +84,9 @@ const ModalCardDetails = ({
   if (sellerId === 'light-up-chinatown') {
     sellerId = '';
     projectId = '1';
+  } else if (sellerId === 'donations-to-apex') {
+    sellerId = '';
+    projectId = '2';
```

Comment:
nit: would be nice to have these as a constant map somewhere. Something like selllerIdToProjectId